src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,30 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderByUserId = async (userId) => {
```

Comment:
Yes i think the it should be user_id, as in the endpoint api/orders?user={user_id}, since you are getting orders for any specific user.